CHANGELOG.md

```diff
@@ -17,6 +19,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed incorrect `if_exists="delete"` handling in `DuckDB.create_table_from_parquet()`
 - Fixed `test_duckdb_to_sql_server.py` tests - revert to a previous version
 - Removed `test__check_if_schema_exists()` test
+- Fixed `sap_rfc.py` source file to not breakdown by both, 
+and extra separator in a row and adding new rows in SAP table between iterations.
```

Comment:
Sorry I don't understand:
- what does "not breakdown by both" mean here?
- the second part I'm having trouble following at all, something is happening between iterations of something?

Reply:
I have improved the file, check the structure and tell me if it is better now.